cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestManager.java

```diff
@@ -404,6 +404,12 @@ private File createModuleJarFile(Class<?> cls) throws IOException {
   }
 
   @Override
+  public void deleteDatasetInstance(NamespaceId namespaceId, String datasetInstanceName) throws Exception {
```

Comment:
I tried to use that on your suggestion but the issue with using DatasetAdmin.drop is  that it just drops the dataset but the instance of the dataset which got added is still there and if some other test case tried to add instance with same name it fails. So, I think we will need to provide a way for tests to delete the instance once they are done. Let me know your thoughts.